ops/terraform/environments/local/local.tf

```diff
@@ -0,0 +1,12 @@
+variable "aou_env" {}
+variable "reporting_dataset_id" {}
+variable "project_id" {}
+variable "credentials_file" {}
+
+module "local" {
+  source               = "/Users/jaycarlton/repos/workbench/ops/terraform/modules/reporting/"
```

Comment:
If you can't make this parameterizable, then please just put in a placeholder value instead like "REPLACE_ME_WITH_FULL_REPORTING_PATH" so this fails very obviously and can be easily/found replaced.

Reply:
The structure here is changing. I had left out an importan tmodule: `workbench`, which rounds up all the subsystems and services as child modules. Then in the client's repo, we have a simple config file with the variables already assigned (b/c it's not for reuse), and it should be able to point straight at this repo.